aiida_optimize/engines/_particle_swarm.py

```diff
@@ -110,8 +110,8 @@ def submit_initialize(self):
         self.fun_global_best = np.inf
         # Initialize the velocities to random number in [-1,1]
         self.velocities = np.zeros((n_parts, n_vars))
-        for line in range(len(self.velocities)):
-            for col in range(len(self.velocities[line])):
+        for line in range(len(self.velocities)):  # pylint: disable=consider-using-enumerate
```

Comment:
Hmm, I tend to agree with `pylint` here: `for line, _ in enumerate(self.velocities)` looks nicer to me. But both are fine, if you feel strongly about it.

Reply:
Agree.